src/lib/components/Commit.js

```diff
@@ -295,7 +295,8 @@ export const CommitModal = (props) => {
 };
 
 export const CommitButton = (props) => {
-  const accountId = useAccountId();
+  const configs = props.config;
+  const accountId = useAccountId(configs && configs.findLast(config => config && config.networkId)?.networkId);
```

Comment:
let's pass `networkId` into `props` instead of `config`.

Reply:
Also `VM` creates `CommitButton`, so we should pass `networkId` from `VM` instance

Reply:
Done ✅